client/my-sites/themes/theme-showcase.jsx

```diff
@@ -390,6 +405,9 @@ class ThemeShowcase extends React.Component {
 					{ 'all' === this.state.tabFilter.key && this.allThemes( { themeProps } ) }
 					{ 'my-themes' === this.state.tabFilter.key && <ThemesSelection { ...themeProps } /> }
 					{ 'trending' === this.state.tabFilter.key && <TrendingThemes { ...themeProps } /> }
+					{ 'fse' === this.state.tabFilter.key && (
+						<FseThemes listLabel={ ' ' } { ...themeProps } />
```

Comment:
I don't think you need the `listLabel` prop, we removed it from `ThemesSelection` when we removed the heading there.